### DISCUSSION TITLE
Documentaion needed

### DISCUSSION THREAD

**Discussion by FreeCreativ** [r0x0r/pywebview#759] [Q&A]
Please add good documentation or manual for beginners.

**Comment by r0x0r** [comment]
What about this?
https://pywebview.flowrl.com/guide/usage.html

**Comment by FreeCreativ** [comment]
I want to create offline application, but the examples and the things I tried all say online like that flowrl.com example of that usage page. I will appreciate if you could explain more. Thank you.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by r0x0r** [comment]
Take a look at the todos app in the examples folder.

**Comment by FreeCreativ** [comment]
Thank you very much, hope you don’t mind if I consult you at roadblocks.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by FreeCreativ** [comment]
I still can't figure out, the offline window setup or must you be online

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by FreeCreativ** [comment]
Thank you very much took a while but I have figured it out and progressed from there. Thanks a lot. I seriously believe this would fix python UI challenge

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by FreeCreativ** [comment]
function app_lists() {
    pywebview.api.list_maker.then(function (lists) {
        document.getElementById('tyu').innerHTML = typeof (lists); or without the typeof function
    })

Then this

def list_maker(self):
    les = JsonHandler()
    lists = les.json_read()
    # lists = les.js_json(li)
    return lists

I am trying to read data from a json file and return the data to the lists variable in JavaScript but it is not showing me anything, I have test run the python function over again but no change .
Although this worked

function generate() {
    let app = get_data('appname');
    let first = get_data('firstname');
    let middle = get_data('middlename');
    let last = get_data('surname');
    clearall();
    pywebview.api.generate(app, first, middle, last).then(function (pass_phrase) {
        let passy = document.getElementById('password')
        passy.value = pass_phrase;
    })
}
and in this scenario I only returned a single variable. Please I would appreciate your help. Thank you
Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
